### ISSUE TITLE
"The package configuration file ... has been edited manually. To avoid overwriting the change, it has been added to the package.json overrides."

### ISSUE THREAD

**Issue by teohhanhui** [jspm/jspm-cli#1468]
Version: 0.17.0-beta.5

Steps to reproduce:

```
jspm install --latest angular2
jspm install
```

Some of the relevant output:

```
ok   The package configuration file
     public/jspm_packages/npm/rxjs@5.0.0-beta.0.json has been edited manually.
     To avoid overwriting the change, it has been added to the package.json
     overrides.
ok   The package configuration file
     public/jspm_packages/npm/angular2@2.0.0-beta.1.json has been edited
     manually. To avoid overwriting the change, it has been added to the
     package.json overrides.
```

**Comment by guybedford** [comment]
And you definitely didn't edit the file manually? Yeah this has come up for me as well sometimes, will investigate further.

**Comment by teohhanhui** [comment]
The file was untouched.

**Comment by salfield** [comment]
This is definitely happening for me with 0.17.0-beta.9. I've tested wiping out the jspm_packages directory, and it happens almost immediately, when reinstalling packages - to ensure I haven't manually edited anything in the package.json.

**Comment by AdamBurgess** [comment]
If I understand it correctly, it checks to see if the file modification times differ by over 2 seconds, and then copies if it does?
That logic could probably be changed to a 'check if over 2 seconds (as a quick check), and if so, check if the hashed value is changed (a slow check)`

**Comment by guybedford** [comment]
Perhaps we just need to change this to 10 seconds, as I doubt users would be editing that soon after install? Hash checking would definitely be a good idea further, but we don't currently provide a hash implementation (this is delegated to the registry itself to handle currently).

**Comment by guybedford** [comment]
This time is entirely due to the time taken to copy the downloaded package from the cache to the jspm_packages folder. This will always be variable, so the better fix here would be to simply "bump" the modified time of the `.jspm-hash` file after doing this copy operation so that it is immediately happening along with the config file save. That should get us within the 2 seconds then fine.

**Comment by guybedford** [comment]
Added in https://github.com/jspm/jspm-cli/commit/7fa0170841055f7908f8059eb8bd6c173bcddeb5. I believe this should fix this.

**Comment by guybedford** [comment]
Released in 0.17.0-beta.10.
